metrics/stats.go

```diff
@@ -69,20 +74,46 @@ var lastPauseNs uint64
 var lastNumGc uint32
 
 var connections int64
+var connectionsSSE int64
+var connectionsGRPC int64
 
 func IncConnection() {
 	atomic.AddInt64(&connections, 1)
 }
 
+func IncConnectionSSE() {
```

Comment:
__[golint]__
* [golint] exported function IncConnectionSSE should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->